lib/plurimath/xml_engine/oga/dumper.rb

```diff
@@ -0,0 +1,96 @@
+module Plurimath
+  module XMLEngine
```

Comment:
Ruby convention of `XmlEngine`?

Reply:
> Ruby convention of `XmlEngine`?

Yes, you are correct, However, I followed the **Oga** file structure to maintain consistency.
But, I can change it if you prefer.

Reply:
Please make the change. Thanks!

Reply:
Bringing other's bad practices into our own is never a good reason 😉